openpype/hosts/blender/api/pipeline.py

```diff
@@ -372,8 +371,11 @@ def ls() -> Iterator:
     called containers.
     """
 
-    for container in lib.lsattr("id", AVALON_CONTAINER_ID):
-        yield parse_container(container)
+    collections = lib.lsattr("id", AVALON_CONTAINER_ID)
+    scene_collections = list(bpy.context.scene.collection.children_recursive)
```

Comment:
I'm not sure casting it to `list` is necessary. It seems to work without.

Reply:
Yes, you 're right, I had in memory this :
```python
"collection_name" in bpy.context.scene.collection.children
```
Is valid but,
```python
collection in bpy.context.scene.collection.children
```
while raise :
`bpy_prop_collection.__contains__: expected a string or a tuple of strings`

but `children_recursive` return list indeed